store/tikv/region_request_test.go

```diff
@@ -56,10 +69,37 @@ func (s *testRegionRequestSuite) SetUpTest(c *C) {
 	s.regionRequestSender = NewRegionRequestSender(s.cache, client)
 }
 
+func (s *testStoreLimitSuite) SetUpTest(c *C) {
```

Comment:
Please add `TearDownSuite` to close the resources you allocated.

Reply:
Addresssed.